source/apps/server/src/graphQL/question.schema.ts

```diff
@@ -0,0 +1,33 @@
+import { Field, Float, ID, Int, ObjectType } from '@nestjs/graphql'
+import { User } from './user.schema'
+import { Topic } from './topic.schema'
+
+@ObjectType()
+export class Question {
+  @Field((type) => ID)
+  id: string
+
+  @Field()
+  topic: Topic
+
+  @Field()
+  topicId: string
```

Comment:
```suggestion

```

You should not provide ID field along with object itself.

This should apply to other field